src/vs/workbench/contrib/search/browser/searchModel.ts

```diff
@@ -2142,6 +2183,16 @@ export class SearchModel extends Disposable {
 		}
 	}
 
+	private onSearchProgressSync(p: ISearchProgressItem, searchInstanceID: string) {
```

Comment:
The non-sync version has `await this._startStreamDelay`, but I guess I can have a flag to call that or not, since I think (?) nobody is looking at the return type.